src/Resources/Locales/en.json

```diff
@@ -116,6 +116,10 @@
         "schemaValidationFailed": {
             "type": "JSON schema validation failed",
             "message": "JSON is invalid. Check syntax matches required schema."
+        },
+        "timeSeriesDatabaseConnectionFetchFailed": {
+            "type": "Time series database connection validation failed",
+            "message": "Data history is not enabled for selected instance or time series database connection could not be verified."
```

Comment:
Can we add in 'the' in a few spots to make it a little smoother? 
`Data history is not enabled for _the_ selected instance or _the_ time series database connection could not be verified.`

Reply:
after content pass it is changed to "Data history connection information could not be found, make sure data history is enabled for this Azure Digital Twins instance and that you have at least '{{permission}}' access to the Azure Data Explorer instance."